src/components/editModal/editModal.css

```diff
@@ -0,0 +1,21 @@
+.edit__modal {
+    width: 37%;
+    min-width: 400px;
+    max-width: 580px;
+    margin-top: 10rem;
+    border-radius: 8px;
+}
+
+.modal__container{
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    background: var(--overlay);
+    z-index: 4000;  
```

Comment:
Suggestion: Can reduce the z-index. I understand the purpose here. z-index can be of a lesser value

Reply:
The sidebar package that I am using has a higher z-index. Anything less than this will go under that sidebar. Hence this value